compiler/rustc_codegen_llvm/src/common.rs

```diff
@@ -223,13 +222,13 @@ impl<'ll, 'tcx> ConstMethods<'tcx> for CodegenCx<'ll, 'tcx> {
         })
     }
 
+    fn zst_to_backend(&self, _llty: &'ll Type) -> &'ll Value {
+        self.const_undef(self.type_ix(0))
```

Comment:
I don't think `i0` is a legal LLVM type? At least not according to https://llvm.org/docs/LangRef.html#integer-type

Reply:
I copied this from elsewhere... :shrug: